presto-product-tests/conf/docker/common/compose-commons.sh

```diff
@@ -16,7 +16,7 @@ function canonical_path() {
 
 source ${BASH_SOURCE%/*}/../../../bin/locations.sh
 
-export HADOOP_MASTER_IMAGE=${HADOOP_MASTER_IMAGE:-'teradatalabs/cdh5-hive:4'}
```

Comment:
after giving it some thought I think we shouldn't introduce DOCKER_IMAGE_VERSION parametrization to the compose files. I wouldn't like the `compose-commons.sh` to grow to another `run-on-docker.sh` (BTW in Warsaw we jokingly call it 'dockto'...). So, if the only reason for introducing the `DOCKER_IMAGE_VERSION` variable is avoiding duplication, I'd say the cost of more complexity in compose-commons might not be worth it.

OTOH Changing the version used from sequence number ('4') to git hash looks like a good change and will enable for seamless search-and-replace the next time we need to update the image version. After replacing, the change can be further validated by running:
```
$ ./presto-product-tests/conf/docker/singlenode/compose.sh config | grep image
    image: teradatalabs/centos6-java8-oracle:jdk1.8.0_92-80a011f
    image: teradatalabs/cdh5-hive:4
    image: teradatalabs/centos6-java8-oracle:jdk1.8.0_92-80a011f
    image: mysql
    image: postgres
    image: teradatalabs/centos6-java8-oracle:jdk1.8.0_92-80a011f
```